deployment/v3/mosip/restart-cron/install.sh

```diff
@@ -28,6 +28,16 @@ function installing_restart-cron() {
 
   echo "To restart Idgenerator service every four hours set the cronJob accordingly, and provide the time like.. ( EX. time: */4 )"
   read -p "Please enter the time(hr) to run the cronjob every day (time: 0-23) : " time
```

Comment:
what is this time neded for?

Reply:
it is needed because of we restarting idgenerator every hour or suppose your want to restart after 4 hours so that the reason will enter the time as per user convenience.